plugins/extractors/postgres/postgres.go

```diff
@@ -0,0 +1,201 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	_ "github.com/lib/pq"
+	"github.com/odpf/meteor/core/extractor"
+	"github.com/odpf/meteor/plugins"
+	"github.com/odpf/meteor/proto/odpf/meta"
+	"github.com/odpf/meteor/proto/odpf/meta/facets"
+	"github.com/odpf/meteor/utils"
+)
+
+type Config struct {
+	UserID   string `mapstructure:"user_id" validate:"required"`
+	Password string `mapstructure:"password" validate:"required"`
+	Host     string `mapstructure:"host" validate:"required"`
+	Database string `mapstructure:"database" default:"postgres"`
+	Exclude  string `mapstructure:"exclude" default:"information_schema,root,postgres"`
```

Comment:
Do we mean that we are allowing user to pass which databases to exclude from extraction and set those hardcoded values of default db as default here in case they don't add anything??

Reply:
Also, user will have to search these default db's himself to exclude them, If at all he needs to exclude some other db's as well. Since if they pass Exclude value as ```db1, db2```. Extractor will only exclude these 2 and not the ```information_schema,root,postgres``` right?

Reply:
Btw, the test and setup looks much cleaner, will implement those with others!!

Reply:
@GrayFlash Yes, this is to allow people to exclude databases. 
Make sense. We can take this into account to add these to the list.